catalog/build_ci/B008.md

```diff
@@ -0,0 +1,8 @@
+### B008
+
+|**Code**           | **B008** |
+|**Title**          | CI/CD agents installation and config should be in version control.|
+|**Description**    | There should exist scripts to setup the CI runner(s) from scratch.|
```

Comment:
https://github.com/qualiticcommunity/qualiticcommunity.github.io/pull/15#discussion_r763623713

Reply:
Please see if that's OK now.